### ISSUE TITLE
Noisy webworker console.log in v3.0.0

### ISSUE THREAD

**Issue by transcranial** [cornerstonejs/cornerstoneWADOImageLoader#268]
[This line](https://github.com/cornerstonejs/cornerstoneWADOImageLoader/blob/master/src/webWorker/webWorker.js#L81) somehow made it into the published v3.0.0 package uncommented, resulting in excessively noisy console logs.

From `dist/cornerstoneWADOImageLoader.min.js`:

```
self.onmessage=function(r){console.log("web worker onmessage",r.data),"initialize"!==r.data.taskType?"loadWebWorkerTask"!==r.data.taskType?n[r.data.taskType]?n[r.data.taskType].handler(r.data,function(e,i){...
```

Perhaps we can conditionally toggle logging using a `debug` flag?

**Comment by swederik** [comment]
Yeah I saw that too. I guess it was published before I pushed or something.
Also need to add a prepublish script so that doesn't happen again.
